scripts/base.py

```diff
@@ -0,0 +1,80 @@
+import os
+import subprocess
+import sys
+import re
+
+ROOT = (
+    subprocess.check_output(["git", "rev-parse", "--show-toplevel"])
+    .decode("ascii")
+    .strip()
+)
+PROJECT_NAME = "Strawberry"
+
+# Files configuration
+
+RELEASE_FILE_NAME = "RELEASE.md"
+RELEASE_FILE = os.path.join(ROOT, RELEASE_FILE_NAME)
+
+PROJECT_TOML_FILE_NAME = "pyproject.toml"
+PROJECT_TOML_FILE = os.path.join(ROOT, PROJECT_TOML_FILE_NAME)
+
+CHANGELOG_FILE_NAME = "CHANGELOG.md"
+CHANGELOG_FILE = os.path.join(ROOT, CHANGELOG_FILE_NAME)
+
+# Git configuration
+
+GIT_USERNAME = "USERNAME_HERE"
+GIT_EMAIL = "EMAIL_HERE"
+
+
+def run_process(popenargs):
+    return subprocess.check_output(popenargs).decode("ascii").strip()
+
+
+def git(popenargs):
+    # avoid decode ascii for the commit message
```

Comment:
what does this mean? :)

Reply:
If we want to use (for example) emojis in the commit message we cannot use `run_process` because the output is converted to ascii using `decode('ascii')`, hence the new utility only for git

Reply:
got it :)

Reply:
I will write a better comment now